worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/StorageManager.scala

```diff
@@ -219,11 +223,46 @@ final private[worker] class StorageManager(conf: RssConf, workerSource: Abstract
   def updateFileInfosInDB(): Unit = {
     fileInfos.asScala.foreach { case (shuffleKey, files) =>
       try {
-        fileInfosDb.put(dbShuffleKey(shuffleKey), PbSerDeUtils.toPbFileInfoMap(files))
-        logDebug("Update DB: " + shuffleKey + " -> " + files)
+        db.put(dbShuffleKey(shuffleKey), PbSerDeUtils.toPbFileInfoMap(files))
+        logDebug(s"Update FileInfos into DB: ${shuffleKey} -> ${files}")
       } catch {
         case exception: Exception =>
-          logError("Update DB: " + shuffleKey + " failed.", exception)
+          logError(s"Update FileInfos into DB: ${shuffleKey} failed.", exception)
+      }
+    }
+  }
+
+  private def reloadAndCleanUserShuffleKeys(db: DB): Unit = {
+    if (db != null) {
+      val itr = db.iterator
+      itr.seek(USER_IDENTIFIER_PREFIX.getBytes(StandardCharsets.UTF_8))
+      while (itr.hasNext) {
+        val entry = itr.next
+        val key = new String(entry.getKey, StandardCharsets.UTF_8)
+        if (key.startsWith(USER_IDENTIFIER_PREFIX)) {
```

Comment:
Should we break when the condition is false?

Reply:
Agree, seems the iterator's keys are sorted in lexicographic order by default.